.github/workflows/flow-release-artifact.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release to NPM
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  release:
+    name: Release
+    runs-on: [self-hosted, Linux, medium, ephemeral]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # pin@v3
+
+      - name: Setup Node 16
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # pin@v3
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org
+
+      - name: Install packages
+        run: npm ci
+
+      - name: Publish to NPM
+        run: npm publish
```

Comment:
Most likely need to explicitly set `--access=public` since org scoped packages are private by default.

Reply:
Added